runners/flink/flink_runner.gradle

```diff
@@ -25,11 +25,29 @@
 
 import groovy.json.JsonOutput
 
+def base_path = ".."
+
+def overrides(versions, type, base_path) {
+  versions.collect { "${base_path}/${it}/src/${type}/java" } + ["./src/${type}/java"]
+}
+
+def all_versions = ['1.12', '1.13', '1.14', '1.15', '1.16', '1.17']
```

Comment:
can we get these values from gradle.properties so keep a single path maintaining the versions? Just use
```
def all_versions = project.flink_versions.split(',')
```

Reply:
Yes, I've done this [here](https://github.com/apache/beam/pull/30988/files#diff-f6fd92149aed337db34a59ef88ca8ed1c7d9aa2371d48e4eb592c36b453eadceR34) :+1: